synchro/__init__.py

```diff
@@ -774,6 +775,8 @@ def __init__(self, kms_providers, key_vault_namespace, key_vault_client=None, **
 class ClientEncryption(Synchro):
     __delegate_class__ = motor.MotorClientEncryption
 
+    create_encrypted_collection = Sync("create_encrypted_collection")
```

Comment:
Does synchro need to be updated to unwrap the synchro database argument and wrap the returned motor Collection here?

Reply:
I do not think so. Is there a failure that leads you to think that this would be necessary?

Reply:
The fact that you're skipping TestAutomaticDecryptionKeys.test_01_simple_create leads me to believe this does not work.